src/containerapp/azext_containerapp/custom.py

```diff
@@ -1202,13 +1203,14 @@ def containerapp_up(cmd,
             _get_registry_from_app(app, source)  # if the app exists, get the registry
         _get_registry_details(cmd, app, source)  # fetch ACR creds from arguments registry arguments
 
-    used_default_container_registry = False
+    force_single_container_updates = False
     if source:
-        used_default_container_registry = app.run_source_to_cloud_flow(source, dockerfile, build_env_vars, can_create_acr_if_needed=True, registry_server=registry_server)
+        force_single_container_updates = app.run_source_to_cloud_flow(source, dockerfile, build_env_vars, can_create_acr_if_needed=True, registry_server=registry_server)
+        app._set_force_single_container_updates(force_single_container_updates)
```

Comment:
Warning: 
Access to a protected member _set_force_single_container_updates of a class 

Please change to `set_force_single_container_updates`

Reply:
Updated.